src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2583,7 +2580,7 @@ public Iterable<DynamicContentItemVariant> getDynamicContentItemVariants(
       DynamicContentItem item) {
     checkHasId(item);
     return new PagedIterable<>(
-        tmpl("/dynamic_content/items/{id}/variants.json").set("id", item.getId()),
+        cbp("/dynamic_content/items/{id}/variants.json").set("id", item.getId()),
```

Comment:
Not in the list provided in [support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination#h_01GNCT8Z7DJ7G4TQ3CFFXQ007Z](https://support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination#h_01GNCT8Z7DJ7G4TQ3CFFXQ007Z)

The endpoint documentation doesn't include CBP either https://developer.zendesk.com/api-reference/ticketing/ticket-management/dynamic_content/#list-items